src/com/raghav/javabasics/SumOfAllNumbersInArray.java

```diff
@@ -0,0 +1,20 @@
+package com.raghav.javabasics;
+
+//Given an array of numbers, write a program to return the sum of all the numbers.
+
+public class SumOfAllNumbersInArray {
+    public static void main(String[] args) {
+        SumOfAllNumbersInArray result = new SumOfAllNumbersInArray();
+        result.sumOfAllNums(new int[]{1,2,3,5,4,6,2});
+        result.sumOfAllNums(new int[]{});
```

Comment:
Will anything be printed when array is empty? Please add appropriate print statements indicating array is empty. This applies to all the other programs as well